pulser/sequence.py

```diff
@@ -215,6 +215,11 @@ def qubit_info(self) -> dict[QubitId, np.ndarray]:
         """Dictionary with the qubit's IDs and positions."""
         return self._register.qubits
 
+    @property
+    def register(self) -> BaseRegister:
+        """Register with the qubit's IDs and positions."""
+        return self._register
```

Comment:
I agree with this addition. I think that `Sequence._register` is being accessed in `_seq_drawer.py`, do you think you could also replace those calls with `Sequence.register`?